wurst/util/EffectUtils.wurst

```diff
@@ -0,0 +1,27 @@
+package EffectUtils
```

Comment:
Simple extension methods of `effect` should be in `handles/Effect.wurst`

Reply:
Don't see any such in the Utils. One requires closures, the last requires MapBounds import, that should not be part of Handles I'd say...

Reply:
No problem I'm happy